### DISCUSSION TITLE
App Crashes and closes.

### DISCUSSION THREAD

**Discussion by tfernanddes** [TheLastGimbus/GooglePhotosTakeoutHelper#335] [I NEED HELP]
Hi, here when I try to use the script, it closes when trying to finding albums....

It just closes and doesn't export any photo. 

The first step I select the "Takeout" folder which has the "Google Fotos" inside

Then I select the recommended output (all files in one folder)

Then I select the recommended setting when it asks about albums.

The app closes on this screen

![image](https://github.com/user-attachments/assets/302e79e7-3818-4547-b2a3-4f05080652d9)

Any help/fix?

Thanks

**Comment by TheLastGimbus** [comment]
ahhh, yeah this seems to be reproducible with really large takeouts...

will unify all of those discussions at #339
